ssas/groups.go

```diff
@@ -143,8 +136,6 @@ func DeleteGroup(id string) error {
 	event := Event{Op: "DeleteGroup", TrackingID: id}
 	OperationStarted(event)
 
-	db := GetGORMDbConnection()
```

Comment:
Nope. It's been a topic of discussion from past teams about moving to a singleton connection pool, but they didn't have concrete reason to do so previously. Our latest performance tests indicate that doing this will help us avoid the "too many connections" error from postgres (along with matching the pattern established in the main BCDA application.)